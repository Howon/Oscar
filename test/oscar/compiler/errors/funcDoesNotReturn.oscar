def add5(x: int) => int = {
  int result = x + 5;
}

def main() => unit = {
  int x = add5(1);
}