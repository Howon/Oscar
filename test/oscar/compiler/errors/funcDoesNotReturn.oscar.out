This function does not return when required